docs/localization/LOCALIZATION.md

```diff
@@ -589,46 +758,45 @@ registerLocaleData(localeAR, 'ar');
 
 @NgModule({
   imports: [
-  LocalizationModule.forRoot({
-    language: 'fr',
-    endPointUrl: 'http://example.com',
-    bundlesOutputPath: config.LOCALIZATION_BUNDLES_OUTPUT,
-    supportedLocales: ['en-EN', 'fr', 'ar'],
-    debugMode: true
-  }),
-  ...
+    LocalizationModule.forRoot({
+      language: 'fr',
+      endPointUrl: 'http://example.com',
+      bundlesOutputPath: config.LOCALIZATION_BUNDLES_OUTPUT,
+      supportedLocales: ['en-EN', 'fr', 'ar'],
+      debugMode: true
+    })
+    // ...
+  ]
 })
-
 ```
 
-By doing this all your translations will be prefixed by the corresponding localization key. That way you can easily map the text to the key.
+This way, all your translations will be prefixed by the corresponding localization key, so you can easily map the text to the key.
 
 Examples:
 
-- [simpleHeader.airline] My Airline
-
-- [simpleHeader.motto] Let's shape the future of travel
+- `simpleHeader.airline - My Airline`
+- `simpleHeader.motto - Let's shape the future of travel`
 
 ## Enable Chrome extension debugging
 
 The Otter framework provides a [Chrome Extension](https://chrome.google.com/webstore/detail/otter-devtools/aejabgendbpckkdnjaphhlifbhepmbne) to help debug the application.
-To enable the communication with the [Otter Devtools](https://chrome.google.com/webstore/detail/otter-devtools/aejabgendbpckkdnjaphhlifbhepmbne) the two following steps are required:
+To enable the communication with the [Otter Devtools](https://chrome.google.com/webstore/detail/otter-devtools/aejabgendbpckkdnjaphhlifbhepmbne), the two following steps are required:
 
-1. Importing the Devtools module into the application AppModule:
+1. Importing the Devtools module into the application module:
 
 ```typescript
 import { LocalizationDevtoolsModule } from '@o3r/localization';
 
 @NgModule({
   imports: [
-    ...,
+    // ...,
     LocalizationDevtoolsModule
   ]
 })
-export class AppModule { }
+export class AppModule {}
 ```
 
-2. The debug message service needs to be activated
+2. The debug message service needs to be activated:
 
 ```typescript
 import { LocalizationDevtoolsMessageService } from '@o3r/localization';
```

Comment:
Now we recommend to do it in the `main.ts` (you can check the showcase application)
to avoid polluting the app.component.ts